<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SC_Grid</class>
 <widget class="QMainWindow" name="SC_Grid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="SC_TableWidget" name="tblData"/>
    </item>
    <item>
     <widget class="QTableWidget" name="tblTotal">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionEdit"/>
   <addaction name="actionRemove"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionFilter"/>
   <addaction name="actionExcel"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/new.png</normaloff>:/img/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/edit.png</normaloff>:/img/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/filter.png</normaloff>:/img/filter.png</iconset>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/trash.PNG</normaloff>:/img/trash.PNG</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/refresh.png</normaloff>:/img/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionExcel">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/excel.png</normaloff>:/img/excel.png</iconset>
   </property>
   <property name="text">
    <string>Excel</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SC_TableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">sc_tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
